h5io_browser/base.py

```diff
@@ -248,13 +321,60 @@ def _open_hdf(filename: str, mode: str = "r", swmr: bool = False) -> h5py.File:
         h5py.File: open HDF5 file object
     """
     if swmr and mode != "r":
-        store = h5py.File(name=filename, mode=mode, libver="latest")
-        store.swmr_mode = True
+        store = h5py.File(name=filename, mode=mode, libver="latest", swmr=False)
+        if not store.swmr_mode:
+            store.swmr_mode = True
```

Comment:
Is this change necessary? The previous code was based on the example from the `h5py` documentation:
https://docs.h5py.org/en/stable/swmr.html#using-the-swmr-feature-from-h5py